tfdml/runtime_adapter/platform_types.h

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Can we get rid of this file if all it does is defining limits that are already available in the STL?